ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-isis-new.md

```diff
@@ -49,7 +49,6 @@ interface Management1
 
 | Interface | Description | Mode | VLANs | Native VLAN | Trunk Group | Channel-Group |
 | --------- | ----------- | ---- | ----- | ----------- | ----------- | ------------- |
-| Ethernet3 | MLAG_PEER_EAPI-LEAF1B_Ethernet3 | *access | *- | *- | *- | 3 |
```

Comment:
Shouldn't the MLAG peer interface be "somewhere"? eos_cli_config_gen so probably just need to adjust types over there?
Could we fail to exercise some of the documentation templates because these are gone ?